src/pages/projects/AddProjectContributorModal.tsx

```diff
@@ -8,13 +8,17 @@ interface AddProjectContributorModalProps {
   toggleModal: () => void;
   addProjectManager?: boolean;
   suggestedProjectManager?: string;
+  initialSearchTerm?: string;
+  indexToReplace?: number;
 }
 
 export const AddProjectContributorModal = ({
   open,
   toggleModal,
-  suggestedProjectManager,
+  suggestedProjectManager = '',
```

Comment:
Will in a later PR look into if the suggested project manager can be a serach when the modal is opened